HomeWork 5/LoggerAplication/LoggerAplication/Action.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace LoggerAplication
+{
+    internal class Action
+    {
+        public static Result StartMethod()
+        {
+            Logger.GetInstance().Log("Satrt Method", LogType.Info);
+
+            return new Result(true);
+        }
+
+        public static Result SkippedLogicInMethod()
+        {
+            Logger.GetInstance().Log("Skipped Logic In Method", LogType.Warning);
+
+            return new Result(true);
+        }
+
+        public static Result IBrokeALogic()
```

Comment:
We can replace all these three method by a single one, that will accept LogType and Message